juniper_official/System/check-vmcores.rule

```diff
@@ -0,0 +1,43 @@
+/*
+ * This rule detects presence of vmcore (Kernel crash) file(s) on the device.
+ * If vmcore is detected, this rule raises a yellow alarm. It will continue
+ * to report yellow alarm till the vmcore is moved from /var/crash into
+ * some other location or some other device.
+ */
+iceberg {
+    topic system {
+        synopsis "Check presence of Kernel vmcore files";
+        description "This rule detects presence of Kernel vmcore files";
+        rule check-vmcores {
+            sensor vmcores-sensor {
+                iAgent {
+                    file vmcores.yml;
+                    table vmcoreTable;
+                    frequency 60m;
+                }
+            }
+            trigger report-kernel-cores {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed.